cherrypy/lib/cptools.py

```diff
@@ -411,13 +411,8 @@ def session_auth(**kwargs):
 
 
 session_auth.__doc__ = (
-    """Session authentication hook.
-
-    Any attribute of the SessionAuth class may be overridden via a keyword arg
-    to this function:
-
-    """ + '\n'.join(['%s: %s' % (k, type(getattr(SessionAuth, k)).__name__)
-                     for k in dir(SessionAuth) if not k.startswith('__')])
+    "Session authentication hook.\n" +  "Any attribute of the SessionAuth class may be overridden via a keyword arg to this function:"
```

Comment:
This change is superseded by #1812, we don't need it in this PR. Also, you don't need an explicit string concatenation in between brackets + there's two many spaces after the operator.